src/components/create-vm-dialog/createVmDialog.tsx

```diff
@@ -1219,7 +1220,7 @@ class CreateVmModal extends React.Component<CreateVmModalProps, CreateVmModalSta
             validate: false,
             vmName: '',
             suggestedVmName: '',
-            connectionName: LIBVIRT_SYSTEM_CONNECTION,
+            connectionName: this.props.systemSocketAvailable ? LIBVIRT_SYSTEM_CONNECTION : LIBVIRT_SESSION_CONNECTION,
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-1917-009b3757-20250904-162346-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)